src/main/resources/META-INF/plugin.xml

```diff
@@ -30,8 +30,7 @@
 
     <actions>
         <action id="CleanupBranches" class="com.ppolivka.plugin.gitcleaner.BranchCleanerAction" text="_Delete Old Branches..." description="Removes unused branches">
-            <add-to-group group-id="Git.Menu"/>
-            <add-to-group group-id="Git.ContextMenu"/>
+            <add-to-group group-id="Git.MainMenu"/>
```

Comment:
Probably you only need to add this line and keep the rest to make it backward/forward compatible.

In the meantime you can use Find Action (https://www.jetbrains.com/help/idea/searching-everywhere.html#find_action) and search for "Delete old branches" to run it.

Reply:
updated. Thank you for your suggestion.

Reply:
@PavlikPolivka could you please make a new release? This would make it appear again in the menu of the latest IntelliJ IDEA 2020.3.

Reply:
Just an observation:
because older versions of IDEs do not find `group-id` `Git.MainMenu`, action is placed under menu `Other`.
And because action is added to 'old' groups as well, it appears on both places :)

Reply:
Interesting. do you have any better solution? I am just a web-developer and do not have too much experience about plug-in. This plug-in is really useful to me so I hope it will be released soon.

Reply:
AFAIK it can only be done by uploading multiple builds that correspond to different versions. I'm not developing plugins either :)

That being said... I might correct myself about `since-build` parameter. I read a bit about plugins and if I understand correctly, versions below that `since-build` use older version of plugin. 
Maybe @PavlikPolivka knows more about that? Updated version does cause error to be logged in IDE for not finding the correct `group-id`. Which is not really an issue, but someone will report it here. I'm sure of it.

Reply:
I agree with the idea. The older Intellij should use older version of the plug in.

Reply:
Sorry, Iwas completely out for a month.

Yes udating since would prevent the new version for old versions. I am fine with doing it. 

Will merge this after new year.